students/tlugosan/lesson03/mailroom.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+""" Automate a script to send a thank you mail, print a report of donor or quit the program."""
+
+table_dictionary = {
+    'Toni Orlando': [150.00, 200.00, 100.00],
+    'Amanda Clark': [1800.00],
+    'Robin Hood': [1234.56, 4500.34, 765.28],
+    'Gina Travis': [523.10, 75.00],
+    'Mark Johnson': [850.00, 20.14]
+}
+
+
+def select_action(actions):
+    """User selects an action by its corresponding order number."""
+    print('What would you like to do?')
+    for i, item in enumerate(actions):
+        print("{}) {}".format(i + 1, item))
+    choice_actions = input('Pick your selection by their corresponding number: ')
+    while True:
+        if not choice_actions.isnumeric() or int(choice_actions) not in range(1, len(actions) + 1):
+            choice_actions = input('Pick your selection by their corresponding number: ')
+        else:
+            break
+    choice_actions_int = int(choice_actions)
+    return choice_actions_int
+
+
+def sending_thank_you():
+    """Lists all the donors or prompts for a name and donation amount to compile the thank you email."""
+    send_to_name = input("Who do you want to send the email to? Type 'list' for a list of all the donors. ")
+    if send_to_name == 'list':
+        display_donor_dictionary(table_dictionary)
+        send_to_name = input("Who do you want to send the email to? Type 'list' for a list of all the donors. ")
+    if send_to_name not in table_dictionary.keys():
+        table_dictionary[send_to_name] = []
+    donation_amount = float(input("What donation amount do you want to thank them for? "))
+    while True:
+        if type(donation_amount) is not float:
```

Comment:
this validation doesn't make sense because it is always going to be a float via line 36

if you want to ensure it is numeric then this is best done with exception handling which will be later in lesson 5.